app/src/main/java/org/eyeseetea/uicapp/CustomApplication.java

```diff
@@ -0,0 +1,27 @@
+package org.eyeseetea.uicapp;
+
+
+import android.app.Application;
+
+import com.github.stkent.bugshaker.BugShaker;
+import com.github.stkent.bugshaker.flow.dialog.AlertDialogType;
+import com.github.stkent.bugshaker.github.GitHubConfiguration;
+
+public class CustomApplication extends Application {
+
+    @Override
+    public void onCreate() {
+        super.onCreate();
+
+        BugShaker.get(this)
+                .setLoggingEnabled(BuildConfig.DEBUG)
+                .setAlertDialogType(AlertDialogType.APP_COMPAT)
+                .setGitHubInfo(new GitHubConfiguration(
+                        "eyeseetea/uicapp",
+                        "96a31419f58357fab95ec80b56a43c9a1868b429",
```

Comment:
@Amejia481 it looks like you're including the OAuth token directly in the code. This could be a security issue as we're leaving in the github history the credentials to act on behalf of EyeSeeTeaBot user. It's a very limited permissions user, but it would be great if we could read this token from elsewhere, assuming for example the presence of a json configuration file that would be placed in the .gitignore file to avoid git to propose it for uploading to the repository. Can you please change it? Later we can talk about how to remove it from the history (it's not very difficult)